### PR TITLE
🚀 Nuget Packaging and Release Workflow

### LINKED ISSUE #17: 📦 Create Nuget Packages with Precompiled Binaries
when publishing a release, the build system should precompile binaries so that users can pull in the dependencies without having to build from source.

additionally, users should be able to verify the integrity of the precompiled binaries so that they can be sure the binaries were compiled by the correct authority

### PR DESCRIPTION
**Pull request by keithrfung**
_🚨 Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository._

### Issue
*Link your PR to an issue*

Fixes #17 

### Description
Create a Nuget Package with precompiled binaries for Net Stardard 2.0

### Testing
The best way to test this temporarily is by rerunning workflows. 

https://www.nuget.org/profiles/ElectionGuard

### CODE REVIEW

File: bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/ElectionGuard.Encryption.csproj [hunk 1]
```diff
@@ -0,0 +1,90 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (keithrfung) [inline bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/ElectionGuard.Encryption.csproj:1]: This is the new project file to look over

File: bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestManifest.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
...:""some-object-id-negative"",""sequence_order"":1}],""electoral_district_id"":""some-geopoltical-unit-id"",""name"":""some-referendum-contest-name"",""number_elected"":1,""object_id"":""some-referendum-contest-object-id"",""sequence_order"":0,""vote_variation"":""one_of_m""},{""ballot_selections"":[{""candidate_id"":""some-candidate-id-1"",""object_id"":""some-object-id-candidate-1"",""sequence_order"":0},{""candidate_id"":""some-candidate-id-2"",""object_id"":""some-object-id-candidate-2"",""sequence_order"":1},{""candidate_id"":""some-candidate-id-3"",""object_id"":""some-object-id-candidate-3"",""sequence_order"":2}],""electoral_district_id"":""some-geopoltical-unit-id"",""name"":""some-candidate-contest-name"",""number_elected"":2,""object_id"":""some-candidate-contest-object-id"",""sequence_order"":1,""vote_variation"":""one_of_m""}],""election_scope_id"":""some-scope-id"",""end_date"":""2021-02-04T17:19:34Z"",""geopolitical_units"":[{""name"":""some-gp-unit-name"",""object_id"":""some-geopoltical-unit-id"",""type"":""unknown""}],""parties"":[{""object_id"":""some-party-id-1""},{""object_id"":""some-party-id-2""}],""start_date"":""2021-02-04T17:19:34Z"",""type"":""unknown""}";
```
Comment (YsFred4) [inline bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestManifest.cs:14]: Probably want this line and line 28 to be consistent.

Comment (keithrfung) [reply bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestManifest.cs:14]: That's fair but this is migrated code with the focus on packaging.

This is the `main` branches version.
https://github.com/microsoft/electionguard-cpp/blob/0bad662f3c6c32f4f7c257b2826d448930736594/bindings/netstandard/ElectionGuard/Tests/ElectionGuard.Tests/TestManifest.cs#L14

Comment (AddressXception) [reply bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestManifest.cs:14]: yeah we need to remove this anyway and can be handled as part of #194 or #179

File: bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestEncrypt.cs [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
         readonly string plaintext = "{\"object_id\":\"some-external-id-string-123\",\"style_id\":\"jefferson-county-ballot-style\",\"contests\":[{\"object_id\":\"justice-supreme-court\",\"ballot_selections\":[{\"object_id\":\"john-adams-selection\",\"vote\":1},{\"object_id\":\"write-in-selection\",\"vote\":1,\"extra_data\":\"some writein value\"}]}]}";
         [Test]
+        [Ignore("Memory Access Violation")]
```
Comment (AddressXception) [inline bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestEncrypt.cs:12]: what's this about?

Comment (keithrfung) [reply bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestEncrypt.cs:12]: These tests have a Memory Access Violation issue. This is an issue for this:
https://github.com/microsoft/electionguard-cpp/issues/193

**Pull request merged**